### ISSUE TITLE
Tagging chain for jsonrpsee log

### ISSUE THREAD

**Issue by jasl** [paritytech/substrate#12131]
### Is there an existing issue?

- [X] I have searched the existing issues

### Experiencing problems? Have you tried our Stack Exchange first?

- [X] This is not a support question.

### Description of bug

This is extracted from https://github.com/paritytech/substrate/issues/11768

![image_2022-07-01_16-43-46](https://user-images.githubusercontent.com/1024162/177115700-67e06361-78ff-4f11-a4a2-7ab47f4b2ad6.png)

It's quite unclear that which chain get a new connection or disconnected

And,` WS error: i/o error: Transport endpoint is not connected (os error 107); terminate connection 6` is the client force disconnect so I propose it move to `WARN` level and give a more clear log

khala-parachain node is based on `polkadot-v0.9.27` Cumulus which still has this behavior, so it might not `tracing` bug

**Comment by bkchr** [comment]
@niklasad1 jsonrpsee is using its own thread pool or? If yes, the problem is that this is missing: https://github.com/paritytech/substrate/blob/915f3c10f22777a1dc49c1ed894525280ce12244/client/service/src/task_manager/mod.rs#L163

Basically forwarding of the current tracing span, that provides these information to which node the log belongs.

**Comment by niklasad1** [comment]
> @niklasad1 jsonrpsee is using its own thread pool or? If yes, the problem is that this is missing: 

No, it uses the tokio runtime from `sc-cli` https://github.com/paritytech/substrate/blob/master/client/rpc-servers/src/lib.rs#L159

**Comment by bkchr** [comment]
> > @niklasad1 jsonrpsee is using its own thread pool or? If yes, the problem is that this is missing:
> 
> No, it uses the tokio runtime from `sc-cli` https://github.com/paritytech/substrate/blob/master/client/rpc-servers/src/lib.rs#L159 (unless I have introduced a bug somewhere)

Ahh yeah that looks right. However, we don't use the substrate task manager facilities. 

We need to be able to pass some custom spawn function. So that we can instrument the future.
